include/zephyr/dt-bindings/clock/stm32f10x_clock.h

```diff
@@ -19,7 +19,7 @@
 
 /** CFGR1 devices */
 #undef MCO1_SEL /* Need to redefine generic F1 MCO_SEL for connectivity line devices. */
-#define MCO1_SEL(val)           STM32_DT_CLOCK_SELECT((val), 0xF, 24, CFGR1_REG)
+#define MCO1_SEL(val)           STM32_DT_CLOCK_SELECT((val), 27, 24, CFGR1_REG)
```

Comment:
According o the refman, I think the right values here are:
```c
#define MCO1_SEL(val)           STM32_DT_CLOCK_SELECT((val), 26, 24, CFGR1_REG)
```
I think this should be fixed in a specific appended commit.